misc/fpp_to_fypp.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+for file in $(find src -type f | grep -Ev 'autogen' | grep -E '\.fpp$'); do
```

Comment:
as nervous as it makes me for 'breaking changing' it seems like we should do this (actually apply it)? thoughts?

Reply:
we could, it wouldn't exactly be a breaking change though. Git usually handles renaming files pretty well. The upside is that vscode's Modern Fortran extension would work a lot better.

Reply:
yeah that's the upside. actually, it's the only upside, but since everyone is using vscode except for me it seems worthwhile. we can probably apply this in a separate PR. it would require toolchain find/replace.

Reply:
I agree. What would need find/replace-ing in the toolchain?